### PR TITLE
Add UBTU-20-010181 for generating audit record for unsuccessful attem…

### PR DESCRIPTION
**Pull request by dexterle**
#### Description:

- Add UBTU-20-010181
- Removes UBTU-20-010302
- Fix remediations and OVAL definitions for Ubuntu

#### Rationale:

- Removes UBTU-20-010302 which no longer exists, and replaces with UBTU-20-010181
- Part of Ubuntu 20.04 DISA STIG v1r9 profile upgrade
- https://github.com/ComplianceAsCode/content/pull/10738

#### Review Hints:

Build the product:
```
./build_product ubuntu2004
```
To test these changes with Ansible:
```
ansible-playbook build/ansible/ubuntu2004-playbook-stig.yml --tags "DISA-STIG-UBTU-20-010181"
```
To test changes with bash, run the remediation section: `xccdf_org.ssgproject.content_rule_audit_rules_kernel_module_loading_delete`

Checkout Manual STIG OVAL definitions, and use software like DISA STIG Viewer to view definitions.
```
git checkout dexterle:add-manual-stig-ubtu-20-v1r9
```

This STIG can be tested with the latest Ubuntu 2004 Benchmark SCAP. For reference, please review the latest artifacts: https://public.cyber.mil/stigs/downloads/

### CODE REVIEW

**Comment by marcusburghardt** [comment]
I know it is in Draft, so it is just a kind remind to update the PR description.

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_delete/ansible/shared.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # disruption = low
 # strategy = configure
 
-{{% if "ol" in product or 'rhel' in product %}}
+{{% if "ol" in product or 'rhel' in product or 'ubuntu' in product %}}
```
Comment (dodys) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_delete/ansible/shared.yml:7]: you should add `multi_platform_ubuntu` to this ansible platforms

**Review by dodys — CHANGES_REQUESTED** [review]
just a small change needed if you want to have ansible remediation

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_delete/ansible/shared.yml (hunk 1 above)
Comment (dexterle) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_delete/ansible/shared.yml:7]: Thank you for the feedback. This was originally added in this commit in the original PR but the commit was modifying too many rules. I'll add this in when I get the chance

https://github.com/ComplianceAsCode/content/commit/e7d99c03246ebc5eacba43156987e3548a6ea73e

Comment (dexterle) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_delete/ansible/shared.yml:7]: Added!

**Review by dodys — APPROVED** [review]
lgtm, thanks!

**Pull request merged**